activity.py

```diff
@@ -63,11 +63,11 @@ def build_toolbar(self):
         toolbar_box.toolbar.insert(stop_button, -1)
         stop_button.show()
 
-    def read_file(self, file_path):
-        self.game.read_file(file_path)
+    # def read_file(self, file_path):
+    #     self.game.read_file(file_path)
 
-    def write_file(self, file_path):
-        self.game.write_file(file_path)
+    # def write_file(self, file_path):
+    #     self.game.write_file(file_path)
```

Comment:
Why comment them out?

Reply:
No need to comment them out then if the function they called didn't do anything and they'll be useful for sure if you want to implement highscores for instance so this helps you save any data at the end of the game and then reload said data when the activity is resumed.

Reply:
Hmm, I will be removing it for now, and later implement the highscore in another PR as this PR is for refactor